tests/serac_thermal_structural_solver.cpp

```diff
@@ -0,0 +1,168 @@
+// Copyright (c) 2019-2020, Lawrence Livermore National Security, LLC and
+// other Serac Project Developers. See the top-level LICENSE file for
+// details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+#include <gtest/gtest.h>
+
+#include <fstream>
+
+#include "coefficients/linear_coefficient.hpp"
+#include "mfem.hpp"
+#include "serac_config.hpp"
+#include "solvers/thermal_structural_solver.hpp"
+
+void   InitialDeformation(const mfem::Vector &x, mfem::Vector &y);
+void   InitialVelocity(const mfem::Vector &x, mfem::Vector &v);
+double InitialTemperature(const mfem::Vector &x);
+
+TEST(dynamic_solver, dyn_solve)
+{
+  MPI_Barrier(MPI_COMM_WORLD);
+
+  // mesh
+  std::string base_mesh_file = std::string(SERAC_SRC_DIR) + "/data/beam-hex.mesh";
```

Comment:
I like this much better than the old way of finding the mesh files!

Reply:
@white238 is to thank for this one!

Reply:
Actually, SERAC_SRC_DIR is a little confusing, as there is a src/ directory in serac, but that's not what it's pointing to.. Maybe SERAC_DIR or SERAC_BASE_DIR or something would be better

Reply:
SERAC_REPO_DIR?

Reply:
Done!